### PR TITLE
doc: allow specifying short slip distance in train-schedule

### PR DESCRIPTION
**Pull request by bougue-pe**
Replace on_stop_signal flag by reception_signal enum

Prepares https://github.com/OpenRailAssociation/osrd/issues/8809

### CODE REVIEW

**Review by flomonster — APPROVED** [review]
LGTM

**Review by Erashin — APPROVED** [review]
Lgtm

File: content/docs/reference/design-docs/timetable/index.en.md [hunk 1]
```diff
@@ -139,12 +139,19 @@ constraint_distribution: MARECO # or LINEAR
 # times are defined as time elapsed since start. Even if the attribute is omitted,
 # a scheduled point at the starting point is infered to have departure=start_time
 # the "locked" flag is ignored by the backend.
-# To force stop-signal on stop's arrival, you can use the "on_stop_signal" flag.
+#
+# To specify signal's state on stop's arrival, you can use the "reception_signal" enum:
+#   - OPEN: arrival on open signal, will reserve resource
+#   - STOP: arrival on stop signal, will not reserve resource and will trigger
+#      safety speed on approach
```
[the remark this answers is not in the document]
Comment (bougue-pe) [reply content/docs/reference/design-docs/timetable/index.en.md]: Added `downstream of the signal` in https://github.com/OpenRailAssociation/osrd-website/compare/2d04e23591baa720e258c7c406ff44ff7dc72cd7..19f3311f5c320c3b7f70a0a5bee5c2f86ee28a98

File: content/docs/reference/design-docs/timetable/index.en.md [hunk 2]
```diff
@@ -139,12 +139,19 @@ constraint_distribution: MARECO # or LINEAR
...
+#   - SHORT_SLIP_DISTANCE_STOP: arrival on stop signal with a short slip distance,
```
[the remark this answers is not in the document]
Comment (bougue-pe) [reply content/docs/reference/design-docs/timetable/index.en.md]: As discussed with you and confirmed with @flomonster and @Erashin : changing for a shorter `SHORT_SLIP_STOP` in https://github.com/OpenRailAssociation/osrd-website/compare/2d04e23591baa720e258c7c406ff44ff7dc72cd7..19f3311f5c320c3b7f70a0a5bee5c2f86ee28a98.
Reasons:
* "short" already bears the notion of distance
* anyway the notion is a bit too complex to be contained in a name (completed comment a bit)

**Review by woshilapin — APPROVED** [review]

**Pull request merged**